lib/pester/version.rb

```diff
@@ -1,3 +1,3 @@
 module Pester
-  VERSION = '0.1.0'
+  VERSION = '0.1.1'
```

Comment:
actually it's kind of a major one because it's not backwards compatible
(the message: option is now different)

On Wed, Apr 1, 2015 at 1:19 PM, Marc Bollinger <notifications@github.com>
wrote:

> In lib/pester/version.rb
> <https://github.com/lumoslabs/pester/pull/5#discussion_r27607182>:
>
> > @@ -1,3 +1,3 @@
> >  module Pester
> > -  VERSION = '0.1.0'
> > +  VERSION = '0.1.1'
>
> To be a noodge, this is technically a minor bump, per semver
> <http://semver.org/>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lumoslabs/pester/pull/5/files#r27607182>.
>